template/tinyvue/src/views/list/search-table/index.vue

```diff
@@ -1,145 +1,8 @@
-                <tiny-form-item :label="$t('searchTable.columns.workname')">
-                  <tiny-input
-                    v-model="filterOptions.workbenchName"
-                    :placeholder="$t('searchTable.form.input')"
-                  ></tiny-input>
-                </tiny-form-item>
-              </tiny-col>
-              <tiny-col :span="4" label-width="100px">
-                <tiny-form-item
-                  :label="$t('searchTable.columns.enablement')"
-                  prop="id"
-                >
-                  <tiny-input
-                    v-model="filterOptions.project"
-                    :placeholder="$t('searchTable.form.input')"
-                  ></tiny-input>
-                </tiny-form-item>
-              </tiny-col>
-              <tiny-col :span="4" label-width="100px">
-                <tiny-form-item :label="$t('searchTable.columns.type')">
-                  <tiny-input
-                    v-model="filterOptions.type"
-                    :placeholder="$t('searchTable.form.input')"
-                  ></tiny-input>
-                </tiny-form-item>
-              </tiny-col>
-            </transition-fade-slide-group>
-          </tiny-row>
+    <Breadcrumb :items="['menu.list.searchTable']" />
 
-          <tiny-row :flex="true" justify="end" class="col">
-            <transition-fade-slide-group>
-              <tiny-col v-if="setCollapse" :span="4" label-width="100px">
-                <tiny-form-item :label="$t('searchTable.columns.study')">
-                  <tiny-input
-                    v-model="filterOptions.address"
-                    :placeholder="$t('searchTable.form.input')"
-                  ></tiny-input>
-                </tiny-form-item>
-              </tiny-col>
-              <tiny-col v-if="setCollapse" :span="4">
-                <tiny-form-item :label="$t('searchTable.form.status')">
-                  <tiny-select
-                    v-model="filterOptions.status"
-                    :options="statusOptions"
-                  ></tiny-select>
-                </tiny-form-item>
-              </tiny-col>
-              <tiny-col :span="4">
-                <tiny-form-item>
-                  <div class="search-btn">
-                    <transition-fade-slide-group>
-                      <tiny-button type="primary" @click="reloadGrid">
-                        {{ $t('searchTable.form.search') }}
-                      </tiny-button>
-                      <tiny-button @click="handleFormReset">
-                        {{ $t('searchTable.form.reset') }}
-                      </tiny-button>
-                    </transition-fade-slide-group>
-                  </div>
-                </tiny-form-item>
-              </tiny-col>
-            </transition-fade-slide-group>
-          </tiny-row>
-        </transition-fade-slide-group>
-      </tiny-form>
-      <div class="bottom-line">
-        <hr />
-      </div>
+    <div class="contain">
```

Comment:
contain 是一个动词，不太合适，建议：

```suggestion
    <div class="search-table-container">
```